app/models/form/sales/questions/leasehold_charges_known.rb

```diff
@@ -0,0 +1,27 @@
+class Form::Sales::Questions::LeaseholdChargesKnown < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "mscharge_known"
+    @check_answer_label = "Monthly leasehold charges known?"
+    @header = "Does the property have any monthly leasehold charges?"
+    @hint_text = "For example, service and management charges"
+    @type = "radio"
+    @answer_options = ANSWER_OPTIONS
+    @page = page
+    @conditional_for = {
+      "mscharge" => [1],
+    }
+    @hidden_in_check_answers = {
+      "depends_on" => [
+        {
+          "mscharge_known" => 1,
+        },
+      ],
+    }
+  end
+
+  ANSWER_OPTIONS = {
+    "1" => { "value" => "Yes" },
+    "0" => { "value" => "No" },
```

Comment:
There's no consistent rule in the codebase for what numbers correspond to No and Yes (normally 0, 1 or 2 in some order), but I think there should be...

Reply:
Yup, we're saving it the way it's expected to be exported